package.json

```diff
@@ -1,58 +1,64 @@
 {
-  "name": "javascript-demo",
+  "name": "js-encryptedfs",
   "version": "0.0.1",
   "author": "Roger Qiu",
-  "description": "JavaScript Demo Project",
+  "description": "Encrypted filesystem written in TypeScript for Node.js",
   "keywords": [
-    "nix",
-    "demo"
+    "filesystem",
+    "encryption"
   ],
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/MatrixAI/JavaScript-Demo.git"
+    "url": "git+https://github.com/MatrixAI/js-encryptedfs.git"
   },
-  "main": "dist/index.node.cjs.js",
-  "module": "dist/index.node.es.js",
+  "main": "dist/encryptedfs.js",
+  "module": "dist/encryptedfs.js",
   "browser": "dist/index.browser.umd.js",
   "scripts": {
-    "test": "ava -v",
-    "rollup": "rollup --config",
+    "develop": "webpack --mode=development --watch & webpack-dev-server --mode=development & wait",
+    "build": "webpack --mode='production'",
+    "test": "jest",
     "dos2unix": "dos2unix dist/*",
-    "chmod": "chmod u+x dist/bin/*",
-    "build": "npm test && npm run rollup && npm run dos2unix && npm run chmod"
+    "chmod": "chmod u+x dist/bin/*"
   },
-  "ava": {
-    "require": [
-      "@babel/register"
-    ]
+  "jest": {
+    "verbose": true
   },
   "engines": {
     "node": ">=8.7.0"
   },
-  "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/node": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
-    "@babel/register": "^7.0.0",
-    "ava": "^0.25.0",
-    "rollup": "^0.65.0",
-    "rollup-plugin-babel": "^4.0.2",
-    "rollup-plugin-commonjs": "^9.1.6",
-    "rollup-plugin-node-builtins": "^2.1.2",
-    "rollup-plugin-node-globals": "^1.2.1",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "virtualfs": "^2.2.0"
-  },
   "dependencies": {
-    "@babel/runtime": "^7.0.0"
+    "immutable": "^4.0.0-rc.12",
```

Comment:
Note that when using webpack... be ware of what are `dependencies` and what are `devDependencies`. Anything that webpack bundles can be put in `devDependencies`. Anything that `webpack` doesn't bundle is `dependencies`.

This is more important for applications than libraries, because for libraries, almost all external packages are `dependencies`, except for things that are explicitly typescript plugins or compile-phase tools.